### PR TITLE
Show VersionCode (build number) on settings screen

### LINKED ISSUE #551: Add bitrise build number to settings screen
So instead of just "1.0" it says "1.0 (3171)" so we can troubleshoot better

### PR DESCRIPTION
**Pull request by beatbrot**
Fixes #551

## Testing and Review Notes

Just open the settings and see the versioncode

## Screenshots or Videos

_(Optional: to clearly demonstrate the feature or fix to help with testing and reviews)_

## To Do

- [x] add “WIP” to the PR title if pushing up but not complete nor ready for review
- [ ] add unit tests
  - optional: consider adding instrumentation (integration/UI) tests
- [x] double check the original issue to confirm it is fully satisfied
- [x] add testing notes and screenshots in PR description to help guide reviewers
- [ ] consider running this branch in a debug simulator and check for memory leak notifications or any warnings
- [ ] request the "UX" team perform a design review (if/when applicable)
- [x] make sure CI builds are passing (e.g.: fix lint and other errors)
- [x] check on the [accessibility](https://mozilla-lockbox.github.io/lockbox-android/accessibility/) of any added UI

### CODE REVIEW

**Comment by beatbrot** [comment]
Not really sure why CI fails, but it doesn't seem to be caused by my changes

**Comment by deandreamatias** [comment]
I have the same problem in #553. 
I also do not think it's because of my changes.

**Comment by beatbrot** [comment]
Oh sorry... I hope I can fix it tomorrow. But if someone else wants to do it until I have time again, I have no issues with that

File: app/src/main/java/mozilla/lockbox/presenter/SettingPresenter.kt [hunk 1]
```diff
@@ -158,7 +160,7 @@ class SettingPresenter(
                 toggleObserver = sendUsageDataObserver
             ),
             AppVersionSettingConfiguration(
-                text = "App Version: $versionNumber",
+                text = "App Version: $versionName ($versionNumber)",
```
Comment (jhugman) [inline app/src/main/java/mozilla/lockbox/presenter/SettingPresenter.kt]: Can we put this string into a resources?

Comment (jhugman) [reply app/src/main/java/mozilla/lockbox/presenter/SettingPresenter.kt]: I think I'd prefer to put the`@StringRes Int` and the version name and number in to the `AppVersionSettingConfiguration` and then combine it in a `setText` method in [`AppVersionSettingViewHolder`][1]

[1]: https://github.com/mozilla-lockbox/lockbox-android/blob/b1ee56c4949d36762c554d84aca259de13de6aaf/app/src/main/java/mozilla/lockbox/view/SettingViewHolder.kt#L104

**Comment by beatbrot** [comment]
I am not really sure how I made these tests fail...maybe someone can help me out?

**Comment by devinreams** [comment]
Thanks for your contribution @beatbrot! I merged in against `master` which had a few more changes and after re-running the flakey/failing test has cleared back up.

I also just re-requested review from the team to look at what you have here. Thanks again! 🙏

**Pull request merged**